includes/Activation_Flag.php

```diff
@@ -73,9 +81,12 @@ public function get_activation_flag( $network_wide = false ) {
 	/**
 	 * Deletes the flag that the plugin has just been activated.
 	 *
-	 * @since 1.10.0 Migrated from Activation class.
+	 * Migrated from Activation class.
```

Comment:
@swissspidy Not sure what this comment was.

Reply:
No idea either 🤷  Probably added by accident